lib/ansible/modules/cloud/azure/azure_rm_virtualmachinescaleset.py

```diff
@@ -178,6 +178,10 @@
         description:
             - Load balancer name.
         version_added: "2.5"
+    app_gateway:
```

Comment:
application_gateway? it's not clear on how to abbreviation

Reply:
yeah, i can change